index.css

```diff
@@ -0,0 +1,83 @@
+.landing-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  flex-direction: column;
+  gap: 1rem;
+  width: 100%;
+  flex-wrap: wrap;
+}
+.main-logo {
+  font-size: 7rem;
+  font-weight: bold;
+  padding: 1rem;
+}
+
+.main-logo::first-letter {
+  color: #9370db;
```

Comment:
maybe you may use color variables here

Reply:
Thank for pointing it out srishti. I will work on it.